### ISSUE TITLE
Documentation?

### ISSUE THREAD

**Issue by johnmiroki** [HydraChain/hydrachain#81]
Hi, I'm as new to HydraChain as I'm to blockchain/Ethereum. I'd like to put hydrachain into use without getting to know the tech details underneath, could someone please point me to some good docs/tutorials? Thanks in advance!

**Comment by TeemoGuo** [comment]
Almost no official docs are there, but I have some material written in Chinese, do you want ? If you know some Chinese, it’s a good idea.

原始邮件
发件人:John Daynotifications@github.com
收件人:HydraChain/hydrachainhydrachain@noreply.github.com
发送时间:2016年11月16日(周三) 09:51
主题:[HydraChain/hydrachain] Documentation? (#81)

Hi, I'm as new to HydraChain as I'm to blockchain/Ethereum. I'd like to put hydrachain into use without getting to know the tech details underneath, could someone please point me to some good docs/tutorials? Thanks in advance!

**Comment by johnmiroki** [comment]
yes, please. Could you please send a copy to fall4her(at)gmail.com? Many
thanks! 非常感谢

TeemoGuo notifications@github.com于2016年11月16日 周三14:29写道：

**Comment by kirushna-kumar** [comment]
@TeemoGuo Hi, can I also get a copy? flipsyde606@gmail.com 
Thanks

**Comment by xyq22222** [comment]
@TeemoGuo  I also want to get a copy, can I ? xuyuqin_sdu@163.com
Thanks

**Comment by KRYPTOHAUS** [comment]
https://github.com/HydraChain/hydrachain/blob/58fae05666b0d919f00b404e09f6cf67966822a1/hydrachain/native_contracts.py

Much of the documentation is somewhat buried.

**Comment by sidraamalik** [comment]
Hi, Did anyone of you manage to get some documentation? please help me getting some

**Comment by charnasol** [comment]
Can you please send a copy to tradingllc2012 at gmail
